### PR TITLE
Add meta data to infrastructure csproj

### PR DESCRIPTION
**Pull request by NullableInt**
This should fix the package name when we publish the `infrastructure` project.

### CODE REVIEW

**Review by asbjornu — APPROVED** [review]

File: src/.nuspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <!-- Required elements-->
+        <id>SwedbankPay.Sdk</id>
+        <description>A client library and SDK for Swedbank Pay's RESTful API Platform</description>
+        <authors>Swedbank Pay</authors>
+        <version>1.1.1</version>
```
Comment (asbjornu) [inline src/.nuspec:8]: Is it possible to not hard-code `version` in the `.nuspec` and instead just provide it on the command line as minted by GitVersion?

Comment (NullableInt) [reply src/.nuspec:8]: Yes, already done

Comment (asbjornu) [reply src/.nuspec:8]: But the `version` element still needs to be in the `.nuspec` file?

Comment (NullableInt) [reply src/.nuspec:8]: Yes, it needs to be there, but it gets overwritten from the CLI. I've tested it locally to ensure it gets overwritten.

Comment (asbjornu) [reply src/.nuspec:8]: Is it possible to set the version number to something invalid so version `1.1.1` is not accidentally published anywhere?

```suggestion
        <version>0.0.1-INVALID</version>
```

File: src/SwedbankPay.Sdk.Infrastructure/SwedbankPay.Sdk.Infrastructure.csproj [hunk 2]
```diff
@@ -3,6 +3,13 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>8.0</LangVersion>
```
Comment (asbjornu) [inline src/SwedbankPay.Sdk.Infrastructure/SwedbankPay.Sdk.Infrastructure.csproj:5]: Shouldn't `LangVersion` be the same in `Core` and `Infrastructure`?

File: src/.nuspec [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <!-- Optional elements -->
+        <projectUrl>https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet</projectUrl>
+        <license type="expression">Apache-2.0</license>
+        <icon>icon.png</icon>
+        <copyright>Swedbank Pay</copyright>
+        <tags>SwedbankPay sdk payments payment client api rest restful</tags>
+        <language>en-US</language>
+    </metadata>
+    <files>
+        <file src="../icon.png" target="" />
```
Comment (asbjornu) [inline src/.nuspec:19]: The `icon` element above references `./icon.png`, while this `file` element references `../icon.png`. Which one is correct? And is it necessary to explicitly include `icon.png` when it is already referenced with `<icon/>`?

File: src/.nuspec [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+        <!-- Optional elements -->
+        <projectUrl>https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet</projectUrl>
+        <license type="expression">Apache-2.0</license>
+        <icon>icon.png</icon>
+        <copyright>Swedbank Pay</copyright>
+        <tags>SwedbankPay sdk payments payment client api rest restful</tags>
+        <language>en-US</language>
+    </metadata>
+    <files>
```
Comment (asbjornu) [inline src/.nuspec:18]: Shouldn't any of the project files or DDLs be referenced by the `.nuspec` file?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: src/.nuspec (hunk 3 above)
Comment (NullableInt) [reply src/.nuspec:19]: Yes, it's necessary. https://docs.microsoft.com/en-us/nuget/reference/nuspec#icon

File: src/.nuspec (hunk 4 above)
Comment (NullableInt) [reply src/.nuspec:18]: I did not find any such parameter to pass inn. And the project file has to be provided when running `dotnet restore`

Comment (asbjornu) [reply src/.nuspec:18]: As long as the published NuGet works, it's fine. Can we add a `nuget publish` and `nuget install` step in GitHub Actions so we know it all works as it should?

Comment (NullableInt) [reply src/.nuspec]: I'll get on that

**Review by asbjornu — APPROVED** [review]

**Pull request merged**